internal/database/database.go

```diff
@@ -2,14 +2,14 @@ package database
 
 import (
 	"context"
-	"os"
 	"fmt"
 	"github.com/jackc/pgx/v5"
 	"log"
-	"file-storage/internal/login"
+	"os"
+	"file-storage/internal/userstruct"
```

Comment:
пакет гитхабный делай сразу, неособо делают так

```suggestion
	"github.com/kgugunava/file-storage/internal/userstruct"
```

только это в го моде нужно настроить